### PR TITLE
Makes electrical toolboxes have random cable colours

### PR DESCRIPTION
**Pull request by BatElite**
[feat]
## About the PR 
This PR makes the cable coils that spawn in electrical toolboxes are of a random colour (3 of one type).

Big cheers to Pali, who actually knew how to do this.
## Why's this needed? 
Makes it easier for crew to get coloured cables and also add some variety in the stuff you throw out of a toolbox.

## Changelog

```
(u)BatElite:
(+)The cables in electrical toolboxes are now of a random colour. Go do aesthetic engineering.
```

### CODE REVIEW

File: code/obj/item/storage/toolbox.dm [hunk 1]
```diff
@@ -98,8 +98,22 @@
 	spawn_contents = list(/obj/item/screwdriver,\
 	/obj/item/wirecutters,\
 	/obj/item/device/t_scanner,\
-	/obj/item/crowbar,\
-	/obj/item/cable_coil = 3)
+	/obj/item/crowbar)
+
+	make_my_stuff()
+		if (!istype(src, /obj/item/storage/toolbox/electrical/mechanic_spawn))
+			spawn_contents[ pick(/obj/item/cable_coil,\
```
Comment (ZeWaka) [inline code/obj/item/storage/toolbox.dm:105]: this is some weird syntax, why not just add to the list?

Comment (BatElite) [reply code/obj/item/storage/toolbox.dm:105]: There's not enough room in the mechanic one for 3 coils, although we could remove a few of the food items. I think those are largely filler?

Comment (ZeWaka) [reply code/obj/item/storage/toolbox.dm:105]: what? i don't care about the number at all, just the code syntax

**Comment by BatElite** [comment]
Since the new method required I add coils separately instead of doing 3 at once I took the liberty of making the single coil in mechanics' boxes random too.

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**